Add Surya model

as-suvorov opened the pull request:
Ticket: CVS-131138

eaidova commented on .ci/ignore_pip_conflicts.txt:
```diff
@@ -14,4 +14,5 @@ notebooks/257-llava-multimodal-chatbot/257-llava-multimodal-chatbot.ipynb # tran
-notebooks/280-depth-anything/280-depth-anything.ipynb # install requirements.txt after clone repo
\ No newline at end of file
+notebooks/280-depth-anything/280-depth-anything.ipynb # install requirements.txt after clone repo
+notebooks/285-surya-line-level-text-detection/285-surya-line-level-text-detection.ipynb # requires python >=3.9
```
should some warning be provided in notebook for python3.8?

aleksandr-mokrov commented on notebooks/285-surya-line-level-text-detection/README.md:
```diff
@@ -0,0 +1,27 @@
+
+#### Table of contents:
+1. Fetch test image.
+1. Run PyTorch inference.
```
Change the nubmering

as-suvorov replied:
This is markdown feature that numbers don't have to be in numerical order: https://www.markdownguide.org/basic-syntax/#ordered-lists. The markdown application will render lists in numerical order anyway.
Or maybe there is another reason for ordering?

aleksandr-mokrov commented on notebooks/285-surya-line-level-text-detection/285-surya-line-level-text-detection.ipynb:
Fix the table of contents: #run-OpenVINO-model doesn't work

aleksandr-mokrov commented on notebooks/285-surya-line-level-text-detection/README.md:
```diff
@@ -0,0 +1,27 @@
+* It is specialized for document OCR. It will likely not work on photos or other images.
+* It is for printed text, not handwriting.
+* The model has trained itself to ignore advertisements.
+* Languages with very different character sets may not work well.
```
Probably it make sense to add next information 

> Text recognition (coming soon)
> Table and chart detection (coming soon)

as-suvorov replied:
I would better not advertise this features as it's unclear when they will be added. It is also unclear if or when we are going to expose them in our notebooks.

as-suvorov commented:
<p>In order to reuse <code>surya-ocr</code> package functions I implemented wrapper classes for ov model. This classes are fully depend on surya package internals. I fixed surya version to prevent possible inconsistency with surya package updates. </p>

---

 <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1695/discussion/#comment-1936171480'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>
